integrationTests/json/vm_test.go

```diff
@@ -334,20 +334,21 @@ func TestTimelocks(t *testing.T) {
 	}
 }
 
-// func TestPromises(t *testing.T) {
-// 	executor, err := am.NewArwenTestExecutor()
-// 	require.Nil(t, err)
-// 	runner := mc.NewScenarioRunner(
-// 		executor,
-// 		mc.NewDefaultFileResolver(),
-// 	)
-// 	err = runner.RunAllJSONScenariosInDirectory(
-// 		getTestRoot(),
-// 		"promises",
-// 		".scen.json",
-// 		[]string{})
-
-// 	if err != nil {
-// 		t.Error(err)
-// 	}
-// }
+func TestPromises(t *testing.T) {
+	t.Skip()
```

Comment:
still not working ?

Reply:
Haven't tried yet, but the contracts will likely need minor changes. This is the next step, though.